### ISSUE TITLE
No longer seems to work

### ISSUE THREAD

**Issue by Slapbox** [airbnb/prop-types-exact#13]
There's no effect (no error) when adding extra props to a component that has `exact` wrapping its propTypes.

We're using prop-types-exact@1.20.0 with react@17.0.2.

See https://github.com/airbnb/prop-types/issues/75

**Comment by Slapbox** [comment]
I added some logging to the built JS file and the function never seems to run.

![image](https://user-images.githubusercontent.com/6835891/176975001-c4271415-3e3f-40f8-9741-5095c52796bb.png)

**Comment by Slapbox** [comment]
`typeof exact` yields `function` as expected. Very confusing.

**Comment by ljharb** [comment]
If you can make a repro case with a jsfiddle or codesandbox, or a repro repo, it'd be much easier to figure this out.

**Comment by Slapbox** [comment]
@ljharb thanks for your reply. I found it does work in a plain create-react-app setup.

That left me mightily confused for a while, until I found that our Babel plugins for PROD vs DEV ended up swapped in a recent commit and it was removing propType checks. The thing that misled me here was that a third-party library we're using continued to do propType checks somehow, leading me to believe propTypes more broadly were working.
